rent_golem/backend/rent_golem/clusters/views.py

```diff
@@ -1,9 +1,40 @@
-from rest_framework import viewsets
+from rest_framework import viewsets, status
+from rest_framework.response import Response
 
-from .models import Clusters
-from .serializers import ClustersSerializer
+from rest_framework.permissions import IsAuthenticated
 
+from .models import Cluster
+from .serializers import ClusterSerializer
 
-class ClustersViewSet(viewsets.ModelViewSet):
-    queryset = Clusters.objects.all()
-    serializer_class = ClustersSerializer
+
+class ClusterViewSet(viewsets.ModelViewSet):
+    permission_classes = [IsAuthenticated]
+    http_method_names = ['get', 'post', 'head', 'patch', 'delete']
```

Comment:
I think we do not need `head` method at the moment?

Reply:
Deleted